website/addons/wiki/utils.py

```diff
@@ -154,3 +156,78 @@ def format_wiki_version(version, num_versions, allow_preview):
         raise InvalidVersionError
 
     return version
+
+def get_wiki_permission(node):
+    """Return if wiki is publicly editable from node
+    :param node: Node to which the wiki belongs
+    :raises: NodeStateError if Node does not have wiki
+    :returns: 'public' if wiki is publicly editable, 'private' if not
+    """
+    wiki = node.get_addon('wiki')
+    if not wiki:
+        raise NodeStateError('Node must have wiki addon.')
+    if wiki.is_publicly_editable:
+        return 'public'
+    return 'private'
+
+def format_data(user, node_ids):
```

Comment:
Rename to something more specific - I suggest `serialize_wiki_settings`, but you're more familiar with it's specific purpose than I am.